openapi.yaml

```
@@ -21550,6 +21538,99 @@ components:
+                    type: string
+                    format: ip
+                    description: >
+                      The IP address.
+                    example: 192.0.2.141
+                    readOnly: true
+                    x-linode-filterable: true
+                    x-linode-cli-display: 1
+                  gateway:
+                    type: string
+                    nullable: true
+                    format: ip
+                    description: >
+                      The default gateway for this address.
+                    example: 192.0.2.1
+                    readOnly: true
+                  subnet_mask:
+                    type: string
+                    format: ip
+                    description: >
+                      The mask that separates host bits from network bits for this address.
+                    example: 255.255.255.0
+                    readOnly: true
+                  prefix:
+                    type: integer
+                    description: >
+                      The number of bits set in the subnet mask.
+                    example: 24
+                    x-linode-filterable: true
+                    readOnly: true
+                  type:
+                    type: string
+                    enum:
+                    - ipv4
+                    - ipv6
+                    - ipv6/pool
+                    - ipv6/range
+                    description: >
+                      The type of address this is.
+                    example: ipv4
+                    readOnly: true
+                    x-linode-filterable: true
+                    x-linode-cli-display: 2
+                  public:
+                    type: boolean
+                    description: >
+                      Whether this is a public or private IP address.
+                    example: true
+                    readOnly: true
+                    x-linode-cli-display: 3
+                  rdns:
+                    type: string
+                    description: >
+                      The reverse DNS assigned to this address. For public IPv4 addresses,
+                      this will be set to a default value provided by Linode if not
+                      explicitly set.
+                    x-linode-filterable: true
+                    x-linode-cli-display: 4
+                    example: test.example.org
+                    nullable: true
+                  linode_id:
+                    type: integer
+                    description: >
+                      The ID of the Linode this address currently belongs to. For IPv4
+                      addresses, this is by default the Linode that this address was
+                      assigned to on creation, and these addresses my be moved using the
+                      [/networking/ipv4/assign](/docs/api/networking/#ips-to-linodes-assign)
+                      endpoint. For SLAAC and link-local addresses, this value may not be changed.
```

```suggestion
                      The ID of the Linode this address currently belongs to. For IPv4
                      addresses, this defaults to the Linode that this address was
                      assigned to on creation. IPv4 addresses may be moved using the
                      [/networking/ipv4/assign](/docs/api/networking/#ips-to-linodes-assign)
                      endpoint. For SLAAC and link-local addresses, this value may not be changed.
```
Bit of a run-on sentence there, and a typo. Is that better?